Source/WebCore/platform/graphics/PlatformDisplay.cpp

```diff
@@ -576,16 +576,16 @@ const Vector<PlatformDisplay::DMABufFormat>& PlatformDisplay::dmabufFormats()
             reinterpret_cast<PFNEGLQUERYDMABUFMODIFIERSEXTPROC>(eglGetProcAddress("eglQueryDmaBufModifiersEXT")) : nullptr;
 
         // For now we only support formats that can be created with a single GBM buffer for all planes.
-        static const Vector<uint32_t> s_supportedFormats = {
+        static const Vector<EGLint> s_supportedFormats = {
+            DRM_FORMAT_ARGB8888, DRM_FORMAT_XRGB8888, DRM_FORMAT_ABGR8888, DRM_FORMAT_XBGR8888,
+            DRM_FORMAT_RGBA8888, DRM_FORMAT_RGBX8888, DRM_FORMAT_BGRA8888, DRM_FORMAT_BGRX8888,
```

Comment:
I wonder if it would make sense to list all the formats with alpha first, and then the ones without: ARGB > ABGR > RGBA > BGRA >> XRGB > XBGR > RGBX > XRGB.

It looks to me that in general we may want to increase the chances of choosing a format with alpha, in case e.g. web views are configured with a non-opaque background color. Or, even better: prefer XRGB variants when the background is opaque, ARGB otherwise... but this would be quite some more work and IMO out of the scope for this one patch 😉